apps/andi/lib/andi_web/live/helpers/form_section.ex

```diff
@@ -15,6 +15,7 @@ defmodule AndiWeb.FormSection do
           socket.assigns.changeset
           |> Map.put(:action, :update)
 
+        AndiWeb.Endpoint.broadcast_from(self(), "form-save", "save-all", %{})
```

Comment:
Are these synchronous enough to not be a problem?

Reply:
I haven't run into any issues with it, but that's a good point